modules/serverless-odc-create-containersource.adoc

```diff
@@ -32,7 +32,7 @@ You can switch between the *Form view* and *YAML view*. The data is persisted wh
 // Optional? Add options and what they mean.
 // Same for env variables...
 .. Optional: In the *Environment variables* field, add any environment variables to set in the container.
-.. In the *Sink* section, add a sink where events from the container source are routed to. If you are using the *Form* view, you can choose from the following options:
+.. In the *Target* section, add a sink where events from the container source are routed to. If you are using the *Form* view, you can choose from the following options:
```

Comment:
Just double-checking that *Target* and "sink" are both intended to coexist in the first sentence here.

Reply:
BTW, you can run `$ grep -rine "sink"` in each relevant directory of the repository to catch any leftovers.

Reply:
Also, I see a few "sink" instances with `Ctrl+F` in https://62975--docspreview.netlify.app/openshift-serverless/latest/eventing/event-sources/serverless-target-source-odc.html.

Reply:
The concepts of "event sinks" are still relevant, so the term is used throughout serverless docs, it's just this one instance in the dev console that is changing literal names from **Sink** -> **Target** in a form 🙂
I question the UX here a bit, because I think it made more sense to just call it **Sink** in the console, but all we can do at this point is make the docs accurate 🤷🏻 😹